tools/check_stamped.py

```diff
@@ -66,7 +73,17 @@ def needStampCheck(filename: str) -> bool:
                 if hasKeySequence(
                         save,
                         "Advanced Micro Devices, Inc. All rights reserved"):
-                    if debug: print("....Already Stamped: Skipping  file ")
+                    yearOfLastCommit = getYearOfLatestCommit(filename)
```

Comment:
You may need to double check that this number isn't getting stuck at 2023. Currently the licensing pass is failing.